src/App.css

```diff
@@ -1,7 +1,52 @@
 body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
   background-color: #6d7c81;
 }
 
 .app-container {
-  padding-top: 30px;
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 20px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #333;
+  color: #fff;
+  padding: 10px;
+}
+
+.navbar-logo {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.navbar-links {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.navbar-links li {
+  margin-right: 10px;
+}
+
+.navbar-links li:last-child {
+  margin-right: 0;
+}
+
+.navbar-links li a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.navbar-links li a:hover {
+  background-color: #555;
+  padding: 5px;
+  border-radius: 5px;
```

Comment:
- [ ] Great job adding stylings to keep the pages good, but kindly follow the layout of the given wireframe. You can add colors, fonts, nominations, etc to the app, but keeping the original design is a must. I have noticed that even the calculator design is different from the one you were given in the previous milestone, is there a reason as to why you did this?

![design](https://github.com/Eccb7/Math-Magicians/assets/53124548/37d23f8a-4ead-4c7e-b725-2a387d170dc6)